dkms.in

```diff
@@ -912,6 +913,11 @@ prepare_signing()
             ;;
         *)
 
+            if [ ! "${sign_file}" ]; then
```

Comment:
```suggestion
            if [ ! -x "${sign_file}" ]; then
```

let's check that it points to an executable file, instead of checking if the string is empty :+1:

Reply:
Done, I also switched to [[ and noticed that we actually need [[ ! -f "$file" || ! -x "$file" ]] here.